test/onnx/gen_onnx.py

```diff
@@ -31,6 +31,7 @@
 
 
 def onnx_test(op_test):
+
```

Comment:
Please make sure to run the same version of yapf as is found on the CI.

Reply:
Yup. Done